commons/src/main/java/org/openmrs/eip/app/management/config/ManagementDataSourceConfig.java

```diff
@@ -68,16 +69,16 @@ public LocalContainerEntityManagerFactoryBean entityManager(final EntityManagerF
 	public PlatformTransactionManager transactionManager(@Qualifier("mngtEntityManager") EntityManagerFactory entityManagerFactory) {
 		return new JpaTransactionManager(entityManagerFactory);
 	}
-	
+
+	// see https://camel.apache.org/manual/camel-4-migration-guide.html#_camel_jpa
 	@Bean(value = "jpa")
 	public JpaComponent jpa(@Qualifier(value = "mngtEntityManager") EntityManagerFactory entityManagerFactory,
 	                        @Qualifier(value = "mngtTransactionManager") PlatformTransactionManager txMgr) {
-		
-		JpaComponent comp = new JpaComponent();
-		comp.setEntityManagerFactory(entityManagerFactory);
-		comp.setTransactionManager(txMgr);
-		
-		return comp;
+		JpaComponent jpaComponent = new JpaComponent();
```

Comment:
I typically prefer variables names that are semantically meaningful in English (rather than connected to their type). In short I prefer to name something "`component`" rather than "`jpaComponent`" unless there is a necessity to be that specific (I don't like "`comp`" either, better a clearly readable name).